apps/nextjs/src/components/AppComponents/Chat/chat-message/index.tsx

```diff
@@ -108,7 +108,7 @@ export function ChatMessage({
                 }}
                 className="underline"
               >
-                content guidance
+                View guidance
```

Comment:
should "view" be capilatised after a colon? 🤷‍♀️

Reply:
I guess it's a button, and we have sentence case on button text!

Reply:
Ah, it's since changed!